crates/libeq_wld/src/parser/fragments/texture.rs

```diff
@@ -52,9 +52,22 @@ impl FragmentParser for TextureFragment {
         let (i, (name_reference, flags, frame_count)) =
```

Comment:
Ooops, looks like this was just assuming there were never any sleep/current_frame values. Exporting to RON/JSON and then re-importing and then doing a binary diff uncovered a lot of little issues like this. I'm sure there are more I haven't run into.